deploy/manifests/dev/us-east-2/tenant/storetheindex/kustomization.yaml

```diff
@@ -18,4 +18,4 @@ secretGenerator:
 images:
 - name: storetheindex
   newName: 407967248065.dkr.ecr.us-east-2.amazonaws.com/storetheindex/storetheindex # {"$imagepolicy": "storetheindex:storetheindex:name"}
-  newTag: 20220914114047-eb96c7e8266dbe8e056e760044754a48e008199b # {"$imagepolicy": "storetheindex:storetheindex:tag"}
```

Comment:
Do not deploy this tag; it adds primary gc changes and being tested on storetheindex-debug